prow/cmd/crier/main.go

```diff
@@ -120,6 +121,21 @@ func (o *options) validate() error {
 		}
 	}
 
+	if o.gcsWorkers > 0 {
+		// use gcsWorkers if blobStorageWorkers is not set
+		if o.blobStorageWorkers == 0 {
+			o.blobStorageWorkers = o.gcsWorkers
```

Comment:
Should this be `+=`? Do we support having gcsWorkers and k8sGCSWorkers both > 0 right now?

Reply:
`+=` is fine for. I would have guessed that either the new or old flags are set and not both.

crier_deployment.yaml suggests it works:
```
        - --gcs-workers=1
        - --kubernetes-gcs-workers=1
```

Reply:
I think we should error if both are set or not? That's not a valid configuration and the user apparently did not understand that one flag replaces the other.

Also, `+= is fine for me. ` no its not. Then we silently enable `k8sgcsworkers` just because someone enabled `gcsworker`.

(As a pure anectdotial sidecoment, I find the format of setting a worker number > 0 in order to enable a controller a super bad UX. We should just provide an option to enable the given controller, set a sane default for workers and allow to optionally override that number. But that has nothing to do with this PR.)

Reply:
Agree that this flag should be a boolean

Also I'm fine with erroring if both are set as that is a bad scenario (but we don't currently check for it)

Reply:
Yup let's keep the current version and error when both old and new flags are set

Reply:
Done